src/components/App/App.css

```diff
@@ -19,6 +19,32 @@
 .body {
   padding: 50px;
   flex-grow: 1;
+  margin: 0 auto;
+  max-width: 480px;
+}
+
+@media screen and (min-width: 820px) {
+  .body {
+    max-width: 720px;
+  }
+}
+
+@media screen and (min-width: 1060px) {
+  .body {
+    max-width: 960px;
+  }
+}
+
+@media screen and (min-width: 1300px) {
+  .body {
+    max-width: 1200px;
+  }
+}
+
+@media screen and (min-width: 1540px) {
+  .body {
+    max-width: 1440px;
```

Comment:
But the issue is 1 max width doesn't look good on all the screens. Especially the small ones. Which is why I did several max-widths. Try it out and resize the browser. It looks good even until the mobile size.